frontend/src/form/FormEventFragments.ts

```diff
@@ -10,9 +10,8 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import gql from 'graphql-tag';
 
-export const subscribersUpdatedEventPayloadFragment = gql`
+export const subscribersUpdatedEventPayloadFragment = `
```

Comment:
The form representation `FormWebSocketContainer` cannot possibly work with the state of this pull request.